### ISSUE TITLE
[The Legend of Zelda: Breath of the Wild] FPS++ (and Static FPS) won't work with v192 v80

### ISSUE THREAD

**Issue by NoUserNameForYou** [ActualMandM/cemu_graphic_packs#523]
On game v192, dlc v80 the fps++ refuses to work, tried all sub settings. Still at 30 FPS.
However, tried an old graphic pack (I believe 708 release) and it works.

**Expected behavior**
Unlocked FPS

**Desktop (please complete the following information):**
 - OS: Windows 10 2004
 - GPU: Nvidia 970
 - Renderer Vulkan
 - Version latest public 1.22.10b

**Comment by Crementif** [comment]
We recently dropped support for V176 and V192 in FPS++ V2.0, which fixed many problems (see https://github.com/ActualMandM/cemu_graphic_packs/commit/ca22977ef03e4a826f3bfe7f0df92447437c4219). I recommend just updating your game to V208.

Before we did this big update, the V192 version of FPS++ was already missing an audio fix. With this update however it would've been missing even more and people would have been confused as to why only some graphic packs are working. Backporting the fixes is too much work for us, but I hope you understand why we dropped support at least.
